### PR TITLE
Update README.md

### LINKED ISSUE #3: docs: Update file path in Readme.md
# Issue 🦠

Under screenshots section we provide links to users, so that they can easily access the code to a specific page. The codebase has now been upgraded but the links are still pointing to the old files, which do not exist at the moment. In this task, we want to update the link to the files.

## Reference ⬆

image

## Example 🐲

```md
FROM: 
https://github.com/AmanNegi/FlutterUIKit/blob/main/lib/pages/auth/login_page.dart

TO:
https://github.com/AmanNegi/FlutterUIKit/blob/main/src/lib/pages/auth/login_page.dart
```

### PR DESCRIPTION
**Pull request by viktoriussuwandi**
fix #3 

Hi @AmanNegi,
just trying to help on this issue, please check it out

### CODE REVIEW

**Comment by AmanNegi** [comment]
Hey @viktoriussuwandi looks great, could you do the same for all the screenshots links?

**Comment by viktoriussuwandi** [comment]
sure @AmanNegi,
just lemme know which `url` should be update & the `new url` one

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Whether you're new to Flutter or looking to enhance your UI design skills, Flutt
...
+|<img width="1604"  src="https://github.com/AmanNegi/FlutterUIKit/blob/main/screenshots/login_page.png">[Login Page](https://github.com/AmanNegi/FlutterUIKit/blob/main/src/lib/pages/auth/login_page.dart) |  <img Width="1604"  src="https://github.com/AmanNegi/FlutterUIKit/blob/main/screenshots/signup_page.png">[SignUp Page](https://github.com/AmanNegi/FlutterUIKit/blob/main/lib/pages/auth/signup_page.dart)|<img width="1604"  src="https://github.com/AmanNegi/FlutterUIKit/blob/main/screenshots/auth_page.png">[Auth Page](https://github.com/AmanNegi/FlutterUIKit/blob/main/lib/pages/auth/auth_page.dart) |
```
Comment (AmanNegi) [inline README.md]: For example, if you need to change the file path, you can easily find it in `lib/pages/{file_name}/{file_name.dart}`.

You can also alternatively open the project in search mode by pressing `T` and find the {file_name.dart}, and get its path easily.

**Review by AmanNegi — COMMENTED** [review]
Sorry, forgot to post the review. Feel free to follow the mentioned.

**Comment by AmanNegi** [comment]
@viktoriussuwandi Any updates on this?

**Comment by viktoriussuwandi** [comment]
Hi @AmanNegi, sorry for late response, I just get back from my college's exam :).

will get it done soon. Thanks for your reminder by the way.

**Comment by AmanNegi** [comment]
No problems @viktoriussuwandi, feel free to work when you are available. 🤗

**Comment by viktoriussuwandi** [comment]
Hi @AmanNegi , sorry for late update,
I think this PR is ready, please do the review again,
just lemme know if any URL goes wrong

**Comment by AmanNegi** [comment]
Amazing work @viktoriussuwandi! Everything works as expected, merging this right away. Thanks a lot for contributing :sparkles: :rocket:

**Pull request merged**